Rakefile

```diff
@@ -6,11 +6,15 @@ task :prod do
   sh "bundle exec rackup -p 8080 --env production"
 end
 
+task :hosted_prod_test do
+  sh("FASTLANE_CI_ERB_CLIENT=1 DEBUG=1 FASTLANE_CI_DISABLE_REMOTE_STATUS_UPDATE=1 FASTLANE_CI_DISABLE_PUSHES=1 FASTLANE_CI_SKIP_RESTARTING_PENDING_WORK=1 bundle exec rackup --host 0.0.0.0 -p 8080 --env development")
```

Comment:
This is interesting, this fails `bundle exec rubocop` for me locally:

```
Rakefile:10:121: C: Metrics/LineLength: Line is too long. [215/120]
```

Anybody got an idea on why that is? Rubocop version is locked, and it should have failed this PR's build?